### DISCUSSION TITLE
Lots of `PackFileCacheStrategy` webpack warnings with imports in `next.config.mjs`

### DISCUSSION THREAD

**Discussion by stefanprobst** [vercel/next.js#30870] [Help]
i am getting a lot of  `PackFileCacheStrategy` webpack warnings when importing modules in `next.config.mjs`. is anybody else seeing this?

see for reproduction: https://github.com/stefanprobst/next-issue-webpack-warning

![next-wp](https://user-images.githubusercontent.com/20753323/140039274-06d66388-c31c-4a3b-b791-febcf0d27417.png)

**Comment by equinusocio** [comment]
Yes same, with a lot of modules. It's similar to https://github.com/vercel/next.js/issues/27650

**Comment by macouella** [comment]
You can temporarily turn off the noise by [controlling the infrastructureLogging output](https://webpack.js.org/configuration/other-options/#level):

```
// next.environment.mjs
export default {
  webpack(config){
    config.infrastructureLogging = {
      level: "error",
    }
  }
}
```

**Reply by ZahraaHawie** [reply]
Hello where I should this in next.config.js? @macouella

**Reply by ap-1** [reply]
> Hello where I should this in next.config.js? @macouella

```ts
/** @type {import("next").NextConfig} */
const config = {
    reactStrictMode: true,
    webpack: (config) => {
        config.infrastructureLogging = {
            level: "error",
        };

        return config;
    },
};

export default config;
```

**Comment by timqian** [comment]
I resolved this issue by removing the `.next` folder and `yarn dev` again

**Reply by ChetSocio** [reply]
Thanks buddy .Really worked for me :)

**Reply by nkpremices** [reply]
Thanks a lot.
That worked for me.

**Reply by jamiehewitt15** [reply]
That worked great, thank you!
